modules/cactus-web/src/Header/Header.tsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types'
+import React, { Children, ReactNode } from 'react'
+import styled from 'styled-components'
+import { margin, MarginProps } from 'styled-system'
+
+import Text from '../Text/Text'
+
+export type BackgroundColorVariants = 'lightContrast' | 'white'
+interface HeaderProps extends MarginProps, React.HTMLAttributes<HTMLDivElement> {
+  bgColor?: BackgroundColorVariants
+  children?: ReactNode
+}
+
+export type HeaderType = React.FC<HeaderProps> & {
+  Item: typeof HeaderItem
+  BreadcrumbRow: typeof HeaderBreadcrumbRow
+  Title: typeof HeaderTitle
+}
+
+export const HeaderItem: React.FC<React.HTMLAttributes<HTMLDivElement>> = ({ children }) => {
+  return <>{children}</>
+}
+
+export const HeaderTitle: React.FC = ({ children }) => {
+  return (
+    <Text as="h1" textStyle="h2" my="0">
```

Comment:
I goofed on the ACs.  Originally I thought this had to render an `h1` tag for accessibility, but it turns out that you can start at any h-level (you just don't want to skip levels).  Therefore, it's probably simpler to change this to the following:

```suggestion
    <Text as="h2" my="0">
```